example_scenes/basic.py

```diff
@@ -125,16 +125,17 @@ def construct(self):
 
         # create dict from list of tuples each having key-mobject pair
         pairs = [("s", square), ("c", circle)]
-        my_dict = VDict(*pairs, show_keys=True)
+        my_dict = VDict(pairs, show_keys=True)
 
         # display it just like a VGroup
         self.play(ShowCreation(my_dict))
         self.wait()
 
         text = TextMobject("Some text").set_color(GREEN).next_to(square, DOWN)
 
-        # add like a VGroup
-        my_dict.add(("t", text))
+        # add like a VGroup, but inside of a list of tuple for now
+        # after attrs branch is merged, it will be easier like `.add(t=text)`
+        my_dict.add([("t", text)])
```

Comment:
Comment: I agree this way of adding things should stay!

Suggestion: we should perhaps also support `my_dict["t"] = text`, so that `VGroup`s are even more like a dictionary. If you don't have time (or don't want to) do that just now, could you please open an issue requesting this feature? Thanks.

Reply:
This feature already exists! There is an example on line [142](https://github.com/nilaybhatia/manim/blob/2507f8ac4839658f6193ca166e0d6d4749f29132/example_scenes/basic.py#L142)

Reply:
Oh forgive me I missed it at first.